doc/modules/reference.rst

```diff
@@ -277,3 +277,31 @@ uses.
    clean
 
 
+.. _connectivity_ref:
+
+:mod:`nilearn.connectivity`: Functional Connectivity
+====================================================
+
+.. automodule:: nilearn.connectivity
+   :no-members:
+   :no-inherited-members:
+
+**Classes**:
+
+.. currentmodule:: nilearn.connectivity.connectivity_matrices
```

Comment:
I am not sure off the top of my head either. There is a bug in sphinx 1.3.1 and we should probably try with sphinx 1.2 before drawing any conclusion. See #712. 

BTW I am going to update circle.yml directly in master.